artview/components/limits.py

```diff
@@ -208,11 +209,11 @@ def limits_dialog(limits, name):
     retval = LimsDialog.exec_()
     print retval
     if retval == 1:
-        limits['vmin'] = float(ent_dmin.text())
-        limits['vmax'] = float(ent_dmax.text())
+        cmap['vmin'] = float(ent_dmin.text())
+        cmap['vmax'] = float(ent_dmax.text())
         limits['xmin'] = float(ent_xmin.text())
         limits['xmax'] = float(ent_xmax.text())
         limits['ymin'] = float(ent_ymin.text())
         limits['ymax'] = float(ent_ymax.text())
 
-    return limits, retval
+    return limits, cmap, retval
```

Comment:
this is because vmin and vmax are no longer part of limits, but rather of Vcmap. The user however don't see the difference (may be he should, may be he should not, I just didn't change that)